conf/app.routes

```diff
@@ -1,5 +1,6 @@
 # microservice specific routes
 
-POST        /employer/:empRef/employment/:nino             @uk.gov.hmrc.individualincomedesstub.controller.EmploymentController.create(empRef: uk.gov.hmrc.domain.EmpRef, nino: uk.gov.hmrc.domain.Nino)
-POST        /:nino/self-assessment/:taxYear                @uk.gov.hmrc.individualincomedesstub.controller.SelfAssessmentController.create(nino: uk.gov.hmrc.domain.Nino, taxYear: uk.gov.hmrc.individualincomedesstub.domain.TaxYear)
-GET         /individuals/nino/:nino/employments/income     @uk.gov.hmrc.individualincomedesstub.controller.EmploymentIncomeController.employments(nino: uk.gov.hmrc.domain.Nino, interval: org.joda.time.Interval)
+POST        /employer/:empRef/employment/:nino                    @uk.gov.hmrc.individualincomedesstub.controller.EmploymentController.create(empRef: uk.gov.hmrc.domain.EmpRef, nino: uk.gov.hmrc.domain.Nino)
+POST        /:nino/self-assessment/:taxYear                       @uk.gov.hmrc.individualincomedesstub.controller.SelfAssessmentController.create(nino: uk.gov.hmrc.domain.Nino, taxYear: uk.gov.hmrc.individualincomedesstub.domain.TaxYear)
+GET         /individuals/nino/:nino/employments/income            @uk.gov.hmrc.individualincomedesstub.controller.EmploymentIncomeController.employments(nino: uk.gov.hmrc.domain.Nino, interval: org.joda.time.Interval)
+GET         /individuals/nino/:nino/self-assessment/income        @uk.gov.hmrc.individualincomedesstub.controller.SelfAssessmentIncomeController.income(nino: uk.gov.hmrc.domain.Nino, taxYearInterval: uk.gov.hmrc.individualincomedesstub.domain.TaxYearInterval)
```

Comment:
I find it difficult to understand to have 2 different formats for taxYear: TaxYear with "2015-16" and TaxYearInterval with "2016".
Maybe there is no need for TaxYearIntervalQueryStringBinder here and you can just pass in (nino: uk.gov.hmrc.domain.Nino, startYear: Int, endYear: Int)
(As there is no real need for validation for this stub endpoint as it is done beforehand)